scripts/build

```diff
@@ -6,16 +6,44 @@ source $(dirname $0)/version
 cd $(dirname $0)/..
 
 mkdir -p bin
-if [ "$(uname)" = "Linux" ]; then
```

Comment:
Don't you need this? Any reason this is removed?

Reply:
nvm, I see you removed for windows build. Should we add this back so that windows/mac command is the same as others?

Reply:
Well, I don't see an issue with having the variable set even if the host isn't using it. Let's say you set `CROSS` to `true` and the host is a `macOS` host. You technically need the `OTHER` flags in order to perform the cross.